tests/E2E/test/clients/common_client.js

```diff
@@ -120,14 +121,32 @@ class CommonClient {
       .waitForExistAndClick(option.replace('%LANG', language).replace('%ID', id))
   }
 
-  open() {
+  async open() {
     if (headless !== 'undefined' && headless) {
-      return this.client.init().windowHandleSize({width: 1280, height: 899});
+      let headless_client = await this.client.init().windowHandleSize({width: 1280, height: 899});
+      await this.enableDownload();
+      return headless_client;
     } else {
       return this.client.init().windowHandleSize({width: 1280, height: 1024});
     }
   }
 
+  /**
+   * Enable download in headless mode
+   */
+  async enableDownload(){
```

Comment:
If this function is only for headless mode ... what about:
- rename it `enableDownloadForHeadlessMode()` to make it very clear
- add a check inside to make sure no one tries to use it without headless mode (see example below)

```
async enableDownload(){
    if( !global.headless) {
        throw new Error('This function should never be called without headless mode');
    }
    ...
```

Reply:
>     * rename it `enableDownloadForHeadlessMode()` to make it very clear
=> done 

>     * add a check inside to make sure no one tries to use it without headless mode (see example below)
> ```
> async enableDownload(){
>     if( !global.headless) {
>         throw new Error('This function should never be called without headless mode');
>     }
>     ...
> ```
=> No need to throw error, calling the function in HeadFull mode will not affect the test.